src/Microsoft.ML.Data/Model/Onnx/SaveOnnxCommand.cs

```diff
@@ -115,26 +140,39 @@ private void GetPipe(IChannel ch, IDataView end, out IDataView source, out IData
 
         private void Run(IChannel ch)
         {
-            IDataLoader loader;
+            IDataLoader loader = null; ;
```

Comment:
>IDataLoader loader = null; ; [](start = 12, length = 28)

Please move inside the `if (_model == null)` condition. You can then remove this `=null` stuff. As well as one of the semicolons. ;)